_maps/RandomRuins/LavaRuins/lavaland_surface_scp_facility.dmm

```diff
@@ -616,6 +616,13 @@
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
 	},
 /area/ruin/unpowered)
+"uS" = (
+/obj/structure/rack,
+/obj/item/gun/ballistic/automatic/pistol/m1911{
+	spawnwithmagazine = 0
```

Comment:
👁️

Reply:
why even have it spawn anything at this point? give it tendril loot instead of a useless pistol. or do this and theres yet another useless waste of space ruin on lavaland like the xeno nest except doesnt even have pizza.